src/sgb/core.cpp

```diff
@@ -935,13 +935,13 @@ void SGB_core::dump_obj(int obj_index) { }
 void SGB_core::dump_bg(int bg_index) { }
 
 /****** Grabs the OBJ palette ******/
-u32* SGB_core::get_obj_palette(int pal_index) { }
+u32* SGB_core::get_obj_palette(int pal_index) { return nullptr; }
```

Comment:
Just NULL for now, until GBE+ moves to C++11 or newer. I don't know when/if that'll ever happen.